internal/configs/virtualserver.go

```diff
@@ -332,8 +338,10 @@ func generateVirtualServerConfig(virtualServerEx *VirtualServerEx, tlsPemFileNam
 	}
 }
 
-func generateUpstream(upstreamName string, upstream conf_v1alpha1.Upstream, isExternalNameSvc bool, endpoints []string, cfgParams *ConfigParams) version2.Upstream {
+func generateUpstream(upstreamName string, upstream conf_v1alpha1.Upstream, isExternalNameSvc bool, endpoints []string, cfgParams *ConfigParams, isPlus bool) version2.Upstream {
 	var upsServers []version2.UpstreamServer
+	lbMethod := generateLBMethod(upstream.LBMethod, cfgParams.LBMethod)
+	slowStart := generateSlowStartForPlus(upstream, lbMethod)
```

Comment:
instead of 
```
		if isPlus {
			s.SlowStart = slowStart
		}
```
I suggest to do the following:
```
slowStart := ""
if isPlus {
  slowStart := generateSlowStartForPlus(upstream, lbMethod)
}
```
this way `generateSlowStartForPlus` will only be invoked for Plus.